### PR TITLE
Test ory oathkeeper rock release

### PR DESCRIPTION
**Pull request by natalian98**

### CODE REVIEW

**Comment by natalian98** [comment]
@cjdcordeiro could you re-run the failed cla check?

File: oci/oathkeeper/image.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 1
+
+release:
+  latest:
+    end-of-life: "2023-06-22T00:00:00Z"
+    edge: 0.40.3
```
Comment (cjdcordeiro) [inline oci/oathkeeper/image.yaml:6]: this needs to be either a revision number or a fully-formed channel. So I think you're missing the `_edge` if your goal is to follow the 0.40.3_edge tag you're building

File: oci/oathkeeper/image.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+upload:
+  - source: "canonical/oathkeeper-rock"
+    commit: 7c09e02d6a23c9b671f557adb2023fed06be7c86
+    directory: .
+    release:
```
Comment (cjdcordeiro) [inline oci/oathkeeper/image.yaml:12]: you're missing the canonical track here - `<version>-base`. This is mandatory as all images must have at least that track, for consistency. See https://github.com/canonical/oci-factory#image-trigger-file (`upload[*].release.<track>`)

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

**Review by cjdcordeiro — APPROVED** [review]
thanks @natalian98.

I had to force rebase this fork and thus why your PR got altered, but the good news is that your build was successful (https://github.com/cjdcordeiro/oci-factory/actions/runs/5056416465/jobs/9236574095?pr=15#step:5:636).

I'm closing this as there's no point merging it to a fork, but you should now feel free to:
1. keep testing other stuff against this fork, or
2. move upstream to canonical/oci-factory 

;)

**Pull request closed without merging**